TestSuite/playbook.py

```diff
@@ -40,14 +40,24 @@ def build(
         if not self.is_test_playbook and readme is not None:
             self.readme.write(readme)
 
-    def create_default_playbook(self):
+    def create_default_playbook(self, name: str = None):
+        """Creates a new playbook with basic data.
+
+        Args:
+            name: The name and ID of the new playbook, default is "sample playbook".
+
+        """
         default_playbook_dir = 'assets/default_playbook'
-        with open(suite_join_path(default_playbook_dir, 'playbook-sample.yml')) as yml:
-            self.build(yml=yaml.safe_load(yml))
+        with open(suite_join_path(default_playbook_dir, 'playbook-sample.yml')) as yml_file:
+            yml = yaml.safe_load(yml_file)
+            if name:
+                yml['id'] = yml['name'] = name
+            self.build(yml=yml)
 
-    def create_default_test_playbook(self):
+    def create_default_test_playbook(self, name: str = None):
```

Comment:
replace with the current default value.
```suggestion
    def create_default_test_playbook(self, name: str = 'SamplePlaybookTest):
```